Homework5/task1/task1.c

```diff
@@ -0,0 +1,128 @@
+﻿#include "C:\Users\Егор\source\repos\SPBU-Homeworks\Stack\stackModule.h"
+#include <stdio.h>
+#include <locale.h>
+#include <malloc.h>
+#include <stdbool.h>
+#include <string.h>
+#include <stdlib.h>
+#define MaxExpressionLength 50
+
+bool isDigit(char symbol) {
+	char allDigits[11] = { "0123456789" };
+	for (int i = 0; i < 10; ++i) {
+		if (symbol == allDigits[i]) {
+			return true;
+		}
+	}
+	return false;
```

Comment:
```suggestion
	return char >= '0' && char <= '9';
```
А ещё есть функция в стандартной библиотеке, называющаяся примерно так же и делающая то же самое.